code/modules/clothing/suits/armor.dm

```diff
@@ -229,7 +229,7 @@
 			if(M == owner)
 				continue
 			owner.Beam(M,icon_state="lightning[rand(1, 12)]",icon='icons/effects/effects.dmi',time=5)
-			M.adjustFireLoss(25)
+			M.adjustFireLoss(10)
```

Comment:
But electrocute_act() stuns. Even if it only stunned, no damage at all, that would be a MASSIVE buff. You're essentially immune to melee unless someone has insulated gloves and a hardsuit.